src/tutorialGame.cpp

```diff
@@ -221,7 +222,8 @@ void TutorialGame::finish()
         sp::ecs::Entity::destroyAllEntities();
         instance->hideAllScreens();
 
-        gameGlobalInfo->startScenario("tutorial.lua");
+        gameGlobalInfo->startScenario(instance->filename);
+        engine->setGameSpeed(1.0);
```

Comment:
This `setGameSpeed` call prevents the reloaded scenario from being paused upon start.